### DISCUSSION TITLE
[787480]Phoenix Wright: Ace Attorney Trilogy 逆转裁判合集123 [本体]

### DISCUSSION THREAD

**Discussion by samduitaken** [pjy612/SteamManifestCache#448] [Share]
`appid:787480`

keys:
```
787481;21ec2f837fe1dfd3cb4acfb74871f9989d4d75fb292e7ab5b902cbc58f577a63
```

[787481_1871295094555173739.zip](https://github.com/pjy612/SteamManifestCache/files/14127707/787481_1871295094555173739.zip)

邮箱：1527482301@qq.com

**Comment by pjy612** [comment]
* 无清单 #26

**Reply by samduitaken** [reply]
改好了 再看看呢

**Reply by pjy612** [reply]
已经好了啊，打上完成标记了。。。 `done`
